File: Unit 7/Unit7A.py
import numpy as np
from scipy import stats

data = np.loadtxt("C:\\Users\\laure\\OneDrive\\Documents\\Purdue\\IN300 - Programming for Data Analysis\\Unit 7\\IN300_Dataset2.txt")

# Print the mean
print("The mean is: ", np.mean(data))

# Print the median
print("The median is: ", np.median(data))

# Get the mode
md = int(stats.mode(data)[0])

# Print the mode
print("The mode is: ", md)

# Print the standard deviation
print("The standard deviation is: ", np.std(data))

File: Unit 7/Unit7A.r
# Set filepath
dataFile <- file.path("C:\\Users\\laure\\OneDrive\\",
                      "Documents\\Purdue\\IN300 - Programming for Data Analysis",
                      "\\Unit 7\\IN300_Dataset2.txt")

# Parse data
data <- scan(dataFile)

# Print the mean
print(paste("The mean is: ", mean(data)))

# Print the median
print(paste("The median is: ", median(data)))

# Print the standard deviation
print(paste("The sd is: ", sd(data)))

# Print the average deviation
print(paste("The average deviation is: ", sum(abs(data - mean(data))) / length(data)))
